rerun

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
```

Comment:
I'm ok with moving it, my plan at one point in the process was maybe use rerun as the entry point to a lot of other things, but I think it's going to become the wrapper around provisioning scripts (which may end up in a different repository). I'm ok with moving it.